backends-velox/src/test/scala/io/glutenproject/execution/VeloxTPCDSSuite.scala

```diff
@@ -57,6 +57,7 @@ class VeloxTPCDSSuite extends WholeStageTransformerSuite {
       .set("spark.sql.sources.useV1SourceList", "avro")
       .set("park.sql.adaptive.enabled", "true")
       .set("spark.gluten.sql.columnar.maxBatchSize", "4096")
+      .set("spark.gluten.shuffleWriter.bufferSize", "4096")
```

Comment:
The key is inconsistent with   GLUTEN_SHUFFLE_WRITER_BUFFER_SIZE for the case-sensitive problem